tests/pytorch/test_nn.py

```diff
@@ -1383,4 +1422,6 @@ def test_twirls():
     test_atomic_conv()
     test_cf_conv()
     test_hetero_conv()
+    test_hetero_linear_layer()
+    test_hetero_embedding()
```

Comment:
L1425-1426 should not be needed.

Reply:
It's fine to put it here, but not necessary

Reply:
Since the argument is not empty for those tests, better to remove it